modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -0,0 +1,129 @@
+from_user=postgres <6>
+
+allDB=("backstage_plugin_app" "backstage_plugin_auth" "backstage_plugin_catalog" "backstage_plugin_permission" "backstage_plugin_scaffolder" "backstage_plugin_search") <7>
+
+for db in ${!allDB[@]};
+do
+  db=${allDB[$db]}
+  echo Copying database: $db
+  PGPASSWORD=$TO_PSW psql -h $to_host -p $to_port -U $to_user -c "create database $db;"
+  pg_dump -h $from_host -p $from_port -U $from_user -d $db | PGPASSWORD=$TO_PSW psql -h $to_host -p $to_port -U $to_user -d $db
+done
+----
+<1> The destination host name, for example, `<db-instance-name>.rds.amazonaws.com`.
+<2> The destination port, such as `5432`.
+<3> The destination server username, for example, `postgres`.
+<4> The source host name, such as `127.0.0.1`.
+<5> The source port number, such as the `<forward-to-port>` variable. 
+<6> The source server username, for example, `postgres`.
+<7> The name of databases to import in double quotes separated by spaces, for example, `("backstage_plugin_app" "backstage_plugin_auth" "backstage_plugin_catalog" "backstage_plugin_permission" "backstage_plugin_scaffolder" "backstage_plugin_search")`.
+
+. Create a destination database for copying the data:
++
+[source,terminal]
+----
+/bin/bash TO_PSW=<destination-db-password> /path/to/db_copy.sh <1>
+----
+<1> The `<destination-db-password>` variable denotes the password to connect to the destination database.
++
+[NOTE]
+====
+You can stop port forwarding when the copying of the data is complete. For more information about handling large databases and using the compression tools, see the link:https://www.postgresql.org/docs/current/backup-dump.html#BACKUP-DUMP-LARGE[Handling Large Databases] section on the PostgreSQL website.
+====
+
+. Reconfigure your `Backstage` custom resource (CR). For more information, see link:{LinkAdminGuide}#proc-configuring-postgresql-instance-using-operator_admin-rhdh[Configuring an external PostgreSQL instance using the Operator]. 
+. Check that the following code is present at the end of your `Backstage` CR after reconfiguration:
++
+[source,yaml]
+----
+# ...
+spec:
+  database:
+    enableLocalDb: false 
+  application:
+  # ... 
+    extraFiles:
+      secrets:
+        - name: <crt-secret> 
+          key: postgres-crt.pem # key name as in <crt-secret> Secret
+    extraEnvs:
+      secrets:
+        - name: <cred-secret> 
+# ...        
+----
++
+[NOTE]
+====
+Reconfiguring the `Backstage` CR deletes the corresponding `StatefulSet` and `Pod` objects, but does not delete the `PersistenceVolumeClaim` object. Use the following command to delete the local `PersistenceVolumeClaim` object:
+
+[source,terminal]
+----
+oc -n developer-hub delete pvc <local-psql-pvc-name>
+----
+where, the `<local-psql-pvc-name>` variable is in the `data-<psql-pod-name>` format.
+====
+
+. Apply the configuration changes.
+
+
+.Verification
+
+. Verify that your {product-very-short} instance is running and contains the migrated data by running the following command:
```

Comment:
These are supposed to be two parts of the verification:

1. verify your instance is running with all the migrated data
2. verify that your local database is disabled

```suggestion
. Verify that your {product-very-short} instance is running and contains the migrated data.

 .Verify that your {product-very-short} instance doesn't contain the local PostgreSQL by running the following command:
```

Reply:
fixed